treeherder/model/models.py

```diff
@@ -571,6 +571,7 @@ class JobDetail(models.Model):
 
     class Meta:
         db_table = "job_detail"
+        unique_together = (("job", "title", "value", "url"),)
```

Comment:
Ahh, you're right.  The docs showed it with the tuple, but then later without.  Simpler is better.  :)